dspace-api/src/main/java/org/dspace/xmlworkflow/state/actions/userassignment/ClaimAction.java

```diff
@@ -115,7 +125,22 @@ public void regenerateTasks(Context c, XmlWorkflowItem wfi, RoleMembers roleMemb
         } else {
             log.info(LogHelper.getHeader(c, "warning while activating claim action",
                                           "No group or person was found for the following roleid: " + getParent()
-                                              .getStep().getId()));
+                                              .getStep().getId()) + ". The item will be sent forward to the next step");
+
+            try {
+                // Copy of org.dspace.xmlworkflow.XmlWorkflowServiceImpl#processNextStep
+                Workflow wf = workflowFactory.getWorkflow(wfi.getCollection());
+
+                Step nextStep = getParent().getStep().getNextStep(ActionResult.OUTCOME_COMPLETE);
+                WorkflowActionConfig nextActionConfig = nextStep.getUserSelectionMethod();
+                ActionResult newOutcome = nextActionConfig.getProcessingAction().execute(c, wfi, nextStep, null);
+
+                xmlWorkflowService.processOutcome(c, c.getCurrentUser(), wf, nextStep, nextActionConfig,
```

Comment:
Should be able to replace `xmlWorkflowService` with `XmlWorkflowServiceFactory.getInstance().getXmlWorkflowService()` (similar to `activate()` method)